packages/app/src/components/nomination/nominationShow.js

```diff
@@ -32,10 +40,64 @@ const NominationShow = ({ match: { params: { id } } }) => {
       </div>
     );
   }
+  
+  const fields = [
```

Comment:
i think we want this to live in FamilyMemberInfo. I think we can just pass the nomination as a prop into FamilyMemberInfo to handle